index.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-unused-vars */
```

Comment:
[Optional] Kindly remove all disabled Linters, Linting helps in making the code more cleaner and professional 👍 

Please check for other occurrences 🙂

Reply:
Okay. Thanks